client/src/components/resources/SharerManageRequestForm.js

```diff
@@ -107,16 +127,28 @@ export default ({ request: defaultRequest }) => {
     return addAlert('Unable to update', 'error')
   }
 
+  const awaitingMtaSteps = []
+  if (state === 'AWAITING_MTA') {
+    if (needsIrb || needsPayment) {
+      awaitingMtaSteps.push(
+        'Review the additional documents submitted by the requests'
+      )
+    }
+    awaitingMtaSteps.push('Sign and upload the requester signed MTA')
```

Comment:
This is the sharer view, should this say `Sign and upload the fully exectuted MTA`?

Reply:
This is the copy from the comps but i do agree maybe @dvenprasad can weigh in?

Reply:
Yeah let's change it.